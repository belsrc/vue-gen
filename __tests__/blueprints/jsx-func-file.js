/* eslint-disable fp-jxl/no-unused-expression, fp-jxl/no-nil */
const genFnJsx = require('./../../src/blueprints/jsx-func-file');

const compName = 'TestComponent';
const className = 'test-component';

describe('genFnJsx', () => {
  test('has the expected ouput structure', () => {
    const actual = genFnJsx(compName, className);

    expect(actual).toMatchSnapshot();
  });
});
