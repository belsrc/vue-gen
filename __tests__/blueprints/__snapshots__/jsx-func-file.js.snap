// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`genFnJsx has the expected ouput structure 1`] = `
"import './styles.scss';

const TestComponent = ({ props, listeners, data }) =>
  <div class={ \`test-component \${ data.class }\` }>
    (ಠ益ಠ)
  </div>

;

TestComponent.name = 'TestComponent';

export default TestComponent;
"
`;
